src/main/java/edu/nextstep/camp/lotto/domain/LottoFactory.java

```diff
@@ -0,0 +1,6 @@
+package edu.nextstep.camp.lotto.domain;
+
+public interface LottoFactory {
+
+    public Lotto createLottoNumbers();
```

Comment:
```suggestion
   Lotto createLottoNumbers();
```
인터페이스는 접근 제어자가 없어도 기본은 public